spec/helpers/records_helper_spec.rb

```diff
@@ -71,5 +71,15 @@ def mock_record(stubs={})
       helper.record_thumbnail(record, @search).should match('<a href="\/records\/1\?search%5Btext%5D=fish">')
     end
 
+    context '#date_parser_for' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.